api/materialized_views/search/outcomeDescriptionSubset.js

```diff
@@ -0,0 +1,28 @@
+let mongodb = require('../../src/utils/mongodb');
+
+async function update(defaultLog) {
+  // All records with non-blank outcomeDescription fields
+  const aggregate = [
+    {
+      $match: {
+        $and: [{ outcomeDescription: { $exists: true } }, { outcomeDescription: { $nin: [''] } }]
+      }
+    }
+  ];
+
+  try {
+    const db = mongodb.connection.db(process.env.MONGODB_DATABASE || 'nrpti-dev');
+    const mainCollection = db.collection('nrpti');
+
+    console.log('Updating outcome_description_subset');
+    aggregate.push({ $out: 'outcome_description_subset' });
+
+    await mainCollection.aggregate(aggregate).next();
+    db.close();
```

Comment:
Don't close the DB here, it causes the API to crash after running this script.   Just remove this line :)